dist/templates/ovnkube-cluster-manager.yaml.j2

```diff
@@ -0,0 +1,192 @@
+# ovnkube-master
+# daemonset version 3
+# starts ovnkube-cluster-manager
+# it is run on the master(s).  Should be used only if interconnect is enabled.
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ovnkube-master
```

Comment:
Should we actually call this `ovnkube-cluster-manager`?

Reply:
yeah the pod and container names that we are picking are here: https://docs.google.com/presentation/d/10qPTbm4gYdKG0kRu7KI8X-dFuR408onX9ii5jYa6YjQ/edit#slide=id.g22a276f021c_1_0 if anyone has strong opinions let us know